src/js/components/visualization/Visualization.jsx

```diff
@@ -93,6 +93,68 @@ class Visualization extends Component {
       typeGroup
     }
   }
+  
+  renderError (error) {
+    const {visualization} = this.props
+    const {name} = visualization
+    let text = error.message
```

Comment:
since we are destructuring everything else, we might as well on this line also.

`let {message} = error`